saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/mSpRL2017App.scala

```diff
@@ -73,4 +73,7 @@ object SpatialOntologyApp extends App {
   val lmCandidateRelations = getCandidateRelations[Token](lmCandidates, spCandidates)
   relations.populate(trCandidateRelations ++ lmCandidateRelations)
 
+
+
+
```

Comment:
@umarmanzoor it's not mine. The last version in my repo is this: https://github.com/Rahgooy/saul/blob/master/saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/mSpRL2017App.scala